spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/CounterController.java

```diff
@@ -53,7 +52,6 @@
 
 	public static final String COUNTER_PREFIX = "counter.";
 
-	@Autowired
```

Comment:
I just pushed a change to this PR to address the missing `MetricRepository`. This reveals that we have no tests for this. I've asked Sabby to create a story.